files/initramfs-tools/arista-convertfs.j2

```diff
@@ -141,14 +142,20 @@ for x in "$@"; do
         SONIC_BOOT_TYPE=warm*|SONIC_BOOT_TYPE=fast*)
             sonic_fast_reboot=true
             ;;
+        systemd.unit=kdump-tools.service)
+            in_kdump=true
+            ;;
     esac
 done
 
 # Check aboot
 [ -z "$aboot_flag" ] && exit 0
 
+# Check kdump
+$in_kdump && exit 0
```

Comment:
>$in_kdump [](start = 0, length = 9)

Could you compare strings instead of run it?

Reply:
Sure I'll change that.